packages/Sortable/src/Sortable.js

```diff
@@ -75,6 +80,7 @@ const Sortable = ({ children, onChange, hasNumbers, hasZebraStripes, onRemove, .
             data-is-dragging-over={snapshot.isDraggingOver ? true : undefined}
             isDraggingOver={snapshot.isDraggingOver}
             ref={provided.innerRef}
+            role="tree"
```

Comment:
You're onto something here... but... it's _not_ a tree.  Lemme dig a little into this.

Reply:
Ok, it's my understanding from the URL in the description that the `<li>` has to be one of these roles, or the screen reader wont work predictably: https://www.w3.org/TR/wai-aria-1.1/#widget_roles